client/utxorpc-haskell-client.cabal

```diff
@@ -0,0 +1,50 @@
+cabal-version: 2.2
+
+-- This file has been generated from package.yaml by hpack version 0.36.0.
+--
+-- see: https://github.com/sol/hpack
+
+name:           utxorpc-haskell-client
```

Comment:
I think you should take `-haskell` out of the name since this is already a Haskell package.